src/lib/components/Collapsible.svelte

```diff
@@ -59,7 +60,7 @@
 </script>
 
 <div
-  data-tid="collapsible-header"
+  data-tid={testId ?? "collapsible-header"}
```

Comment:
can it be simplified by declaring a default value?

```
export let testId = "collapsible-header";
```

Reply:
oh, yes, thanks!